### PR TITLE
doc(chart): Updated chart readme

### PR DESCRIPTION
**Pull request by ams0**
**Description**:

A more descriptive Readme for the Helm chart

**Affected area**:

- New Functionality      [ ]
- Documentation          [X]
- Install                [ ]
- Control Plane          [ ]
- CLI Tool               [ ]
- Certificate Management [ ]
- Networking             [ ]
- Metrics                [ ]
- SMI Policy             [ ]
- Security               [ ]
- Tests                  [ ]
- CI System              [ ]
- Performance            [ ]
- Other                  [ ]

Please answer the following questions with yes/no.

- Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?

No

### CODE REVIEW

**Comment by ams0** [comment]
@nojnhuh I know correctly entered the comments in the `values.yaml` and let `helm-docs` generate the readme. I did not actually included the readme as I assume the CI will take care of it?

**Review by snehachhabria — APPROVED** [review]

File: charts/osm/values.yaml [hunk 1]
```diff
@@ -71,14 +117,18 @@ OpenServiceMesh:
...
     ## for all proxies in the mesh.
+    # -- Toggles Envoy's tracing functionality on/off for all sidecar proxies in the cluster      
     enable: true
 
     # Tracing destination cluster
+    # -- Tracing destination cluster (must contain the namespace) 
     address: "jaeger.osm-system.svc.cluster.local"
 
     # Destination port for the listener
+    # -- Destination port for the listener
```
Comment (michelleN) [inline charts/osm/values.yaml:128]: This looks like a duplicate of the line above

File: charts/osm/values.yaml [hunk 2]
```diff
@@ -71,14 +117,18 @@ OpenServiceMesh:
...
+    # -- Tracing destination cluster (must contain the namespace) 
     address: "jaeger.osm-system.svc.cluster.local"
 
     # Destination port for the listener
+    # -- Destination port for the listener   
     port: 9411
 
     # Destination's API or collector endpoint where the spans will
     # be sent to
+    # -- Destination's API or collector endpoint where the spans will be sent to
```
Comment (michelleN) [inline charts/osm/values.yaml:133]: Also duplicate

**Review by michelleN — CHANGES_REQUESTED** [review]
Thank you @ams0 ! These changes look great. Some of the comments seem like they are duplicates of the line above it. If you could remove the lines above that are duplicates, that would be great. Will merge as soon as that is done.

**Comment by ams0** [comment]
thanks @michelleN ! Also, to be addressed perhaps later, all values are prefixed with `OpenServiceMesh`, perhaps rendering it at all superfluous.

File: charts/osm/README.md.gotmpl [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+```console
+helm repo add osm https://openservicemesh.github.io/osm
+helm repo update
+```
+
+## Install Chart
+
+```console
+helm install [RELEASE_NAME] osm/osm
```
Comment (michelleN) [inline charts/osm/README.md.gotmpl:21]: Should this be `charts/osm`?

**Review by michelleN — APPROVED** [review]
Had a minor question about the README but I'll go ahead and merge this and we can address the install line in a follow up PR.

File: charts/osm/README.md.gotmpl (hunk 3 above)
Comment (ams0) [reply charts/osm/README.md.gotmpl:21]: No, it seems that's how it looks in the chart repo:

```
~> helm search repo osm
NAME   	CHART VERSION	APP VERSION	DESCRIPTION
osm/osm	0.6.1        	v0.6.1     	A Helm chart to install the OSM control plane o...
```

**Review by nojnhuh — APPROVED** [review]
> @nojnhuh I know correctly entered the comments in the `values.yaml` and let `helm-docs` generate the readme. I did not actually included the readme as I assume the CI will take care of it?

We actually don't have this automated in CI currently, but we have it documented as a manual step as part of cutting releases. I'm not too worried about everything being perfectly in sync between releases. Thanks for digging in to figure out the `helm-docs` details.

**Review by ksubrmnn — APPROVED** [review]

**Pull request merged**